homeassistant/components/media_extractor/__init__.py

```diff
@@ -43,14 +55,62 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+EXTRACT_MEDIA_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_URL): cv.string,
+        vol.Optional(ATTR_FORMAT_QUERY, default=DEFAULT_STREAM_QUERY): cv.string,
```

Comment:
Should this default to the `CONF_DEFAULT_STREAM_QUERY` instead? Otherwise it's weird that the `CONF_DEFAULT_STREAM_QUERY` influences the `play_media` service but not the new service.